pkgs/applications/misc/sweethome3d/editors.nix

```diff
@@ -1,5 +1,17 @@
-{ lib, stdenv, fetchcvs, makeWrapper, makeDesktopItem, jdk, jre, ant
-, gtk3, gsettings-desktop-schemas, sweethome3dApp }:
+{ lib
+, stdenv
+, fetchcvs
+, makeWrapper
+, makeDesktopItem
+# sweethome3d 6.4.2 does not yet build with jdk 9 and later.
+# this is fixed on trunk (7699?) but let's build with jdk8 until then.
+, jdk8
+# it can run on the latest stable jre fine though
+, jre
```

Comment:
is jre used anywhere here or could it get removed?

Reply:
yes, it's used in the wrapper. This way the installed program only relies on the default jre (openjdk15 currently) and not on jdk8 (which is only used for building):

```
$ which sweethome3d-furniture-editor /nix/store/jmpgliklvawhq48v2a370y9y4wiysz7d-FurnitureLibraryEditor-1.19/bin/sweethome3d-furniture-editor
$ nix-store -q --tree /nix/store/jmpgliklvawhq48v2a370y9y4wiysz7d-FurnitureLibraryEditor-1.19 | grep openjdk
+---/nix/store/pcybcbgwm9bwaccj1wdrm6c0axfpc2iv-openjdk-15.0.1-ga
|   +---/nix/store/pcybcbgwm9bwaccj1wdrm6c0axfpc2iv-openjdk-15.0.1-ga [...]
```